### PR TITLE
fix wording on settings

### PR DESCRIPTION
**Pull request by joshblum**

### CODE REVIEW

**Comment by buoyad** [comment]
does setting this still work / actually do the right thing?

File: shared/actions/settings.tsx [hunk 1]
```diff
@@ -287,9 +287,9 @@ function* refreshNotifications() {
         ],
       },
       {
-        description: 'Disable sending/receiving typing notifications',
+        description: 'Display typing notifications',
         name: 'disabletyping',
-        subscribed: !!chatGlobalSettings.settings[`${ChatTypes.GlobalAppNotificationSetting.disabletyping}`],
+        subscribed: !!!chatGlobalSettings.settings[`${ChatTypes.GlobalAppNotificationSetting.disabletyping}`],
```
Comment (hyperobject) [inline shared/actions/settings.tsx]: I think this can just be `!`, not `!!!`

Comment (joshblum) [reply shared/actions/settings.tsx]: oops forgot to push taht

**Comment by joshblum** [comment]
@buoyad yes

**Comment by joshblum** [comment]
you can compare to `keybase chat notification-settings`

File: shared/actions/settings.tsx (hunk 1 above)
Comment (jakob223) [reply shared/actions/settings.tsx]: but what if it's, like, really really exciting

**Pull request merged**